packages/calcite-components/src/components/stepper-item/stepper-item.tsx

```diff
@@ -358,13 +363,14 @@ export class StepperItem
   };
 
   private renderIcon(): VNode {
-    const path = this.selected
-      ? "circleF"
-      : this.error
-      ? "exclamationMarkCircleF"
-      : this.complete
-      ? "checkCircleF"
-      : "circle";
+    const path =
+      this.selected && (this.singleViewMode ? !this.error && !this.complete : true)
```

Comment:
This is getting unwieldy. It might be time to use an if/else for this.

Also, I think you can replace

```ts
this.singleViewMode ? !this.error && !this.complete : true
```

with 

```ts
!this.singleViewMode || (!this.error && !this.complete)
```